auth_signup_verify_email/controllers/main.py

```diff
@@ -34,7 +34,7 @@ def passwordless_signup(self, values):
             values["email"] = values.get("login")
 
         # Remove password
-        values["password"] = False
+        values["password"] = ""
```

Comment:
Well Odoo 10.0 doesn't allow to signup without password (False) but it work with an empty password/string ("").